environments.toml

```diff
@@ -1,97 +1,86 @@
-[development]
-# If given `run-locally = true`, `network` will be launched with:
-#
-#     soroban network start local
-#
-# You can use a shorthand version for `local`:
-#
-#     network = "local"
-network = { name = "local", run-locally = true }
+### Development environment configuration
+[development.network]
+rpc-url = "http://localhost:8000"
+network-passphrase = "Soroban Development Network ; September 2021"
 
-accounts = [
-  # If you provide a string, an identity with this name will be created with
-  # `soroban keys`, which saves the mnemonic in `.soroban/identity`.
-  "alice",
-  
-  # Alternatively, you can provide an expanded definition.
-  #   - `name`: the only required field, same behavior as string value above.
-  #   - `default`: whether to use this account as the `--source` for commands
-  #     that need one. See `init` below. Can only have one default.
-  #   - `mnemonic`: a BIP39 mnemonic for the account. If not provided, a new
-  #     keypair will be generated and saved in the `.soroban/identity` directory.
-  { name = "me", default = true, mnemonic = "boots and cats and boots and cats..." },
-]
-default-account = "me"
+# The rpc-url and network-passphrase are required for all environments.
+# Coming soon: If given `run-locally = true`, the `network` will be launched with:
+# `stellar network container start local`
 
-[development.contracts]
-# Auto-build, -deploy, -bind, and -import all contracts in Cargo workspace
-# Only supported in development & test environments
-cargo-workspace = true
+[[development.accounts]]
+name = "me"
+default = true
 
-# Instead of `cargo-workspace`, you can list each contract individually.
-# `workspace = true` indicates that this project is part of the local cargo
-# workspace. In dev & test environments, workspace contracts will be
-# automatically built, deployed, initialized, bound, and imported.
-soroban-atomic-swap-contract = { workspace = true }
-soroban-auth-contract = { workspace = true }
-soroban-errors-contract = { workspace = true }
-soroban-hello-world-contract = { workspace = true }
-soroban-increment-contract = { workspace = true }
+# You can provide an expanded definition for each account:
+#   - `name`: the only required field, an identity with this name will be created
+#      with `stellar keys`.
+#   - `default`: whether to use this account as the `--source` for commands
+#     that need one. Only one default is allowed. The keypair will be generated
+#      and saved in the `.soroban/identity` directory.
+# Coming Soon: # If you provide just a string, an identity with this name will be created with
+# `soroban keys`, which saves the mnemonic in `.soroban/identity`. E.g.: "alice"
 
-# If contracts need more settings, you can list them the long way.
-[development.contracts.soroban-token-contract]
-workspace = true
+[development.contracts]
+example-core = { workspace = true }
+example-status-message = { workspace = true }
```

Comment:
Just noting that

1. this is perfect for what we currently need, and for use with the almost-ready `loam init`
2. but it will not actually work with `stellar contract init` since these contracts aren't even in the `--with-example` options
3. hard-coding these is worse than what the `soroban-astro-template` currently does 😖

Reply:
(so I'm looking forward to us implementing the `cargo-workspace` setting described on line 38!)